README.md

```diff
@@ -118,9 +118,9 @@ mocks.
 
 ```bash
 $ mockingbird install \
-  --project Bird.xcodeproj \
-  --targets BirdModels BirdManagers \
-  --destination UnitTestTarget
+  --project MyProject.xcodeproj \
+  --targets MyProject SomeExternalTarget \
+  --destination MyProjectTests
```

Comment:
Thanks for the PR! I agree that it could be confusing since most iOS devs might not have multi-module projects. To keep the example more concrete, what do you think about making the example something similar to:

```swift
--project Bird.xcodeproj \
--targets Bird BirdManagers \
--destination BirdTests
```

Reply:
That works! Just the fact that the naming convention is consistent across the three different arguments is much clearer than before. This is the issue I ran into last night. I was attempting to use my BirdModel file, which had a BirdModel protocol and I used that as a target.